### PR TITLE
Header footer fixup

### PR DESCRIPTION
**Pull request by gozart1**
This fixes the logo not showing.
It also removes the header and logo related css from nexthome.css.
So the heights of header and footer are back to their previous dimensions. 
The footer markup has been reworded a bit so that it reflows better.
The logo now collapses to just the symbol for narrow screens.
screen shot 2018-11-01 at 4 16 43 am
Also, there's now a white version of logo for night mode and sim/test net modes. 
screen shot 2018-11-01 at 4 02 53 am
screen shot 2018-11-01 at 4 02 42 am
screen shot 2018-11-01 at 4 05 04 am

There's definitely room for refinement here. But step one was just making it look not broken and I think this achieves that.

### CODE REVIEW

**Comment by buck54321** [comment]
This looks great. Is the text in the logo black or "#212529"?

File: views/extras.tmpl [hunk 1]
```diff
@@ -34,64 +34,63 @@
     <script src="/js/humanize.js"></script>
     <script src="/js/stimulus.umd.js"></script>
     <script src="/js/mousetrap.min.js"></script>
-    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-    <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
-      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
```
Comment (chappjc) [inline views/extras.tmpl]: eeek, I can't believe this slid in.... ugh

**Comment by dmigwi** [comment]
This stuff looks great @gozart1  nice work.

**Comment by buck54321** [comment]
@gozart1 There is one `.navbar-fixed-bottom` left in nexthome.css. Should I move that to main.css?

**Pull request merged**